packages/amplify-cli/package.json

```diff
@@ -66,7 +66,7 @@
     "@aws-amplify/amplify-util-mock": "5.9.4",
     "@aws-amplify/amplify-util-uibuilder": "1.14.3",
     "@aws-cdk/cloudformation-diff": "~2.68.0",
-    "amplify-codegen": "4.7.3",
+    "amplify-codegen": "4.8.0",
```

Comment:
nit: `amplify-codegen` is bumped here but not in the `yarn.lock` as its version was fixed in the root package json resolution. 
https://github.com/aws-amplify/amplify-cli/blob/dev/package.json#L178-L179

Reply:
We do not have any codegen changes that needs to be shipped for Gen1 CLI, let's take care of it when we need to ship new codegen.